### ISSUE TITLE
Inventory is empty and characters are naked

### ISSUE THREAD

**Issue by Blueeyer27** [suriyun-production/mmorpg-kit-docs#300]
When building clean MMO demo project (latest package) on mobile device all characters become naked on the map and inventory is empty. Can't see the other players gear also. Based on tests happened on both iOs and Android devices. On PC version everything worked fine. Maybe I've forgot to enter somewhere server LAN address on the client side. It was changed only in:
1) MMOClientInstance -> MmoNetworkSetting
2) CentralNetworkManager - Network Address (tried both cases, with and without changing it)
3) GameInstance -> NetworkSetting (tried both cases, with and without changing it)

Both mobile devices and server machine were under the same network, machine LAN address was 192.168.1.103, iOs device 192.168.1.102 if that matters.
Machine OS: MacOS Catalina 10.15.1
DB: tried both SQLite and MySQL, running on the same server machine.

Also caught NullReferenceException, idk if it's related to this bug.
PC version (from Unity Editor):
![Screenshot_2020-03-29_at_23 49 39](https://user-images.githubusercontent.com/8277219/77861710-3b242a80-721f-11ea-88ea-40feeeed1f77.jpg)

iOs version:
![image0](https://user-images.githubusercontent.com/8277219/77861711-424b3880-721f-11ea-8ac5-d5ecc6c5efc0.png)

Exception:
![Screenshot_2020-03-30_at_00 17 07](https://user-images.githubusercontent.com/8277219/77861725-63ac2480-721f-11ea-9cbb-75c749e6c2a4.jpg)

**Comment by Blueeyer27** [comment]
iOs build settings (tried both 00Init_MMO and 00Init_MMO_ClientOnly scenes):
![Screenshot 2020-03-30 at 00 49 01](https://user-images.githubusercontent.com/8277219/77861866-56dc0080-7220-11ea-9066-db478c61a16e.jpg)

**Comment by insthync** [comment]
Hello, Try find `NetDataReaderExtension` and changes codes at line 115:
from 
```
                object instance = Reflection.CreateInstanceWithExpression(type);
```
to
```
                object instance = Activator.CreateInstance(type);
```

Then rebuild the iOS build.

**Comment by Blueeyer27** [comment]
@insthync thanks, works like a charm. I guess it should be included in next version.

**Comment by JestHWas** [comment]
@insthync  will this be in the new release? As this also affects android

**Comment by insthync** [comment]
Yes, it will be included in next version

**Comment by insthync** [comment]
This was fixed in 1.54c, if issues still occurring, you can re-open it.
